mypy/checkmember.py

```diff
@@ -91,10 +92,24 @@ def analyze_member_access(name: str, typ: Type, node: Context, is_lvalue: bool,
         if isinstance(ret_type, TupleType):
             ret_type = ret_type.fallback
         if isinstance(ret_type, Instance):
-            result = analyze_class_attribute_access(ret_type, name, node, is_lvalue,
-                                                    builtin_type, not_ready_callback, msg)
-            if result:
-                return result
+            if not isinstance(node, (OpExpr, ComparisonExpr)):
```

Comment:
`node` is just used as a context for errors, and isn't necessarily reliable. I don't think it's the best idea to use it for this purpose.